src/store/actionTypes.ts

```diff
@@ -0,0 +1,7 @@
+export const successAction = (action: string): string => `${action}_SUCCESS`;
```

Comment:
Suggestion: For better typings you can try to use [this](https://www.typescriptlang.org/docs/handbook/2/template-literal-types.html) syntax of template strings types